cvat/apps/quality_control/default_settings.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+
+QUALITY_CHECK_JOB_DELAY = int(os.getenv('CVAT_QUALITY_CHECK_JOB_DELAY', 15 * 60))
+"The delay before the next quality check job is queued, in seconds"
```

Comment:
If we define settings only in the global settings override file (such as `base.py` etc.):
- the code will fail at some point in runtime if any settings are not defined (the application code will depend on the global configuration)
- developers won't be able to determine the list of required settings unless by reading all the code (including the programmatic way, if needed)
- we won't be able to provide default values for settings (or there is going to be special treatment for undefined settings in the implementation, which may become duplicated) or modify the set of settings without breaking something

I wanted to avoid this. The default values still can be overridden in base.py and other settings files, and it is done for the sever api tests.